backend/src/packages/orders/order.entity.ts

```diff
@@ -0,0 +1,142 @@
+import { type IEntity } from '~/libs/interfaces/interfaces.js';
+import { type NullableProperties } from '~/libs/types/types.js';
+
+import { type OrderEntity as OrderEntityT } from './libs/types/types.js';
+
+class OrderEntity implements IEntity {
+  private id: OrderEntityT['id'] | null;
+
+  private price: OrderEntityT['price'];
+
+  private scheduledTime: OrderEntityT['scheduledTime'];
+
+  private startPoint: OrderEntityT['startPoint'];
+
+  private endPoint: OrderEntityT['endPoint'];
+
+  private status: OrderEntityT['status'];
+
+  private userId: OrderEntityT['userId'];
+
+  private businessId: OrderEntityT['businessId'];
+
+  private driverId: OrderEntityT['driverId'];
+
+  private customerName: OrderEntityT['customerName'] | undefined;
+
+  private customerPhone: OrderEntityT['customerPhone'] | undefined;
```

Comment:
```suggestion
  private customerName: OrderEntityT['customerName'];

  private customerPhone: OrderEntityT['customerPhone'];
```
In another comment I mentioned that customerName should be defined, but nullable `| null`

Reply:
fixed